File: Software Technologies/Software-Technologies-Exam-Preparation-Kanban-Board/C#/TeisterMask/Models/Task.cs
using System.ComponentModel.DataAnnotations;

namespace TeisterMask.Models
{
    public class Task
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Title{ get; set; }

        [Required]
        public string Status{ get; set; }

        public Task() { }
        public Task(string title, string status)
        {
            Title = title;
            Status = status;
        }
    }
}

File: Software Technologies/Software-Technologies-Exam-Preparation-Kanban-Board/Java/src/main/java/teistermask/entity/Task.java
package teistermask.entity;

import javax.persistence.*;

@Entity
@Table(name = "tasks")
public class Task {
	private int id;
	private String title;
	private String status;

	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public int getId() {
        return this.id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Column(name = "title", nullable = false)
    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    @Column(name = "status", nullable = false)
    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Task() {}
    public Task(String title, String status){
	    this.title = title;
	    this.status = status;
    }
}
